woocommerce/storefront
# 🐛 FIX: Remove PHPUnit dependency (#1400)

Fixes #1400

Removed the PHPUnit dependency from https://github.com/woocommerce/storefront/compare/fix/%231400?expand=1#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780 and generated new https://github.com/woocommerce/storefront/compare/fix/%231400?expand=1#diff-1da2c7edc898c70e5a79a9997c98cecc

### Screenshots

n/a

### How to test the changes in this Pull Request:

1. Open up https://github.com/woocommerce/storefront/compare/fix/%231400?expand=1#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780 and search for `PHPUnit`
2. This PR works as expected, if no results are found.

### Changelog

> Fix - Remove unused PHPUnit dependency. #1400
